walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -224,8 +224,10 @@ private static boolean attends(Event event, MeetingRequest request) {
                     break;
                 }
             }
-        }
 
-        return included;
+            return included;
+        } else {
```

Comment:
I believe what Arun meant was to change line 211 to be:

if (event == null) {
  return false;
}

And then you can un-indent Collection<String>... up to the end of the function, since there is no need for "else" (since the true branch returns immediately).

Reply:
Ah I see, I was still in the mindset that the "included" variable could be changed somehow. I just made the change.